code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -43,6 +43,10 @@
 	var/obj/machinery/doomsday_device/DOOM = new /obj/machinery/doomsday_device(src)
 	doomsday_device = DOOM
 	verbs -= /mob/living/silicon/ai/proc/nuke_station
+	for(var/obj/item/weapon/pinpointer/point in world)

```

Comment:
In world is already used in existing code for shunting and in malf code before it was removed.